src/components/AddCourseButton.vue

```diff
@@ -0,0 +1,106 @@
+<template>
+  <div
+    class="semester-courseWrapper semester-addWrapper"
+    :class="{ 'semester-addWrapper--compact': compact, 'my-2 mx-0': shouldClearPadding }"
+    @click="onClick"
+    data-intro-group="pageTour"
+    data-step="3"
+    data-intro='<b>Add your course in this semseter!</b><br>
+      <div class = "introjs-bodytext">To start planning your college career, you should try adding a course in your current semester.</div>'
+    data-disable-interaction="1"
```

Comment:
I'm sus about this walkthrough code being here considering this is a reusable component

Reply:
Ooooo yea that is such a good point I forgot about that. I can just remove the props when it's the model from the req menu